.devcontainer/oncreate

```diff
@@ -6,3 +6,6 @@ echo "Running Azure SDK for Rust devcontainer post create script..."
 
 echo "Updating toolchains..."
 rustup show
+
+echo "Installing test-proxy..."
+dotnet tool update azure.sdk.tools.testproxy --global --prerelease --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json --ignore-failed-sources
```

Comment:
This isn't necessary. The test proxy will be acquired automatically and the EXE installed by this tool doesn't use the name we launch, so this step is basically pointless.

Reply:
The intent here was to support running `test-proxy push -a .../assets.json`, but I can remove it if you'd prefer otherwise

Reply:
I realized that above in my comments. I see you removed it and the `dotnet` devcontainer feature. I'd put them back. I wasn't thinking about that originally, but it makes sense.

Reply:
Done!